src/site/content/en/measure/index.njk

```diff
@@ -1,166 +1,99 @@
-          </div>
-        </div>
-
-        <!-- ACCESSIBILITY -->
-        <div class="w-measure-category">
-          <img
-            src="/images/icons/a11y.svg"
-            aria-describedby="lh-category-accessibility-title"
-          />
-          <div
-            class="w-headline--three w-mt--std"
-            id="lh-category-accessibility-title"
-          >
-            Accessibility
-          </div>
-          <div>
-            Checks for common issues that may prevent users from accessing your
-            content.
-          </div>
-        </div>
-      </div>
+  <section class="flow all-center wrapper gap-top-size-4" data-size="narrow">
+    <h2 class="no-link w-headline--two w-font-weight--medium">
+      How your site is measured
+    </h2>
+    <p class="color-mid-text">
+      When you measure your site, web.dev uses
+      <a href="https://developers.google.com/web/tools/lighthouse/">Lighthouse</a>,
+      an open-source, automated tool for improving the quality of web pages.
+      Lighthouse will audit your site in the following categories:
+    </p>
+  </section>
 
-      <div>
-        All tests are run using a simulated mobile device, throttled to a fast
-        3G network &amp; 4x CPU slowdown.
-      </div>
-    </section>
-  </div>
+  <section class="wrapper region measure-faq" class="bg-mid-bg" data-size="narrow">
+    <details class="bg-mid-bg">
+      <summary>
+        What is the data source powering the Measure tool?
+      </summary>
+      <p>
+        Measure tool uses Lighthouse data from  the PageSpeed Insights API.
+        Learn more about PageSpeed Insights in the
+        <a href="https://developers.google.com/speed/docs/insights/v5/about?hl=en-US&utm_source=PSI&utm_medium=incoming-link&utm_campaign=PSI#faq)">
+        official FAQs</a>
+      </p>
+    </details>
+    <details class="bg-mid-bg">
+      <summary>
+        How does Measure tool differ from PageSpeed Insights?
+      </summary>
+      <p>
+        Measure tool  measures the overall quality of a web page in a
+        <a href="https://web.dev/lab-and-field-data-differences/">lab
+        environment</a> (using Lighthouse). It gives insights about multiple
+        quality categories from Lighthouse, including Performance, Accessibility
+        and SEO. PageSpeed Insights (PSI) is performance focused,
+        using both lab (Lighthouse) and field (Chrome UX Report) data
+        to provide a summary of performance.
+        Furthermore, PSI provides origin level insights in the field,
+        whereas web.dev/measure is intended to deep-dive into a single
+        URL in the lab.
+      </p>
+      <p>
+        Use PSI when you want to measure and optimize your Core Web Vitals, and use web.dev/measure when optimizing a broader set of quality signals about a page.
+      </p>
+    </details>
 
+  </section>
 </div>
-
-<code id="guide-audit-mapping" style="display: none;">
```

Comment:
Can we keep this? This helps guide users through the result? Or does the PSI endpoint not provide this?

Reply:
No, sadly all connection between scores and guidance was arbitrarily dropped <sad face>